### PR TITLE
Create CODE_OF_CONDUCT.md

### LINKED ISSUE #19: Add code of conduct file
- Code of conduct file is missing in the project. I would like to add one.

### PR DESCRIPTION
**Pull request by tejan-singh**
fixes #19

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# Contributor Covenant Code of Conduct
```
Comment (sudo-rgorai) [inline CODE_OF_CONDUCT.md]: Please change this to just `Code of Conduct`

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+help@sanscript.tech
+All complaints will be reviewed and investigated promptly and fairly.
+
+All community leaders are obligated to respect the privacy and security of the
+reporter of any incident.
+
```
Comment (sudo-rgorai) [inline CODE_OF_CONDUCT.md]: You can remove the `Enforcement` part for now because at present there is no mechanism for the same.

**Comment by tejan-singh** [comment]
Made requested changes.

**Comment by sudo-rgorai** [comment]
Great! LGTM :+1:

**Pull request merged**